app/assets/javascripts/pieces.js

```diff
@@ -2,5 +2,11 @@ $(document).ready(function(){
   $("td").click(function(){
```

Comment:
@lindyrp got you. The thing is that oftentimes it looks like leftovers from our old features. Also, do you want the users to open the browser console and see all the internals? 😉 

Even no need to stash it. You can always look back in the commits before and restore it or add it from scratch again